### DISCUSSION TITLE
Matomo, GA4, TaC et Angular

### DISCUSSION THREAD

**Discussion by OlivierVS31** [AmauriC/tarteaucitron.js#983] [General]
Bonjour,

Petite question si vous avez déjà eu le cas et du coup quelles sont vos best practices : nous avons un site sous angular. Nous déployons Matomo via TaC que nous avons en version payante. Nous avions déjà ce process pour Google Analytics Universal.

Le déploiement se fait sans problème :) mais nous rencontrons un problème que nous n'avions pas avec Universal, à savoir que les titres des pages ne sont pas remontées. C'est valable pour GA4 comme pour Matomo.

Quelqu'un a t il déjà ce cas et du coup via l'intégration via TaC, comment l'avez vous contourné ?

Merci d'avance

**Comment by AmauriC** [comment]
Bonjour : )

Avec Matomo il y a un paramètre `sendData`, en le mettant à false, vous pouvez gérer l'envoi des pages à Matomo sans passer par tarteaucitron qui ne s'occupera que de charger la librairie Matomo.

**Reply by OlivierVS31** [reply]
Merci je vais essayer de trouver ce paramètre :)

**Reply by OlivierVS31** [reply]
Du coup @AmauriC j'ai pas trouvé mais je garde le tips
